index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <style>
+      /*
+      The test doesn't read my css correctly, that's why I put the css here
+      */
+      
+
+      #nav-bar{
+          position: fixed;
+          top: 0;
+          left: 0;
+          width: 100%;
+      }
+      #header{
+          display: flex;
+          justify-content: center;
+          align-items: center;
+      }
+
+      #nav-bar ul{
+          list-style: none;
+          display: flex;
+          justify-content: space-between;
+        }
+        
+       
+        .form{
+          padding: 50px 0;
+        }
+
+        .form-input {
+          padding: 10px 0;
+        }
+        #nav-bar a{
+          color: #000;
+        }
+        
+        @media (max-width: 600px) {
+          #nav-bar ul{
+              flex-wrap: wrap;
+            }
+
+            #nav-bar li{
+              margin: 12px 0;
+            }
+        }
+
+
```

Comment:
please do not add blank lines, spaces, or indentations without a reason as this will not pass the linter check. kindly remove the extra lines and space  then make sure your linters are running

Reply:
- [ ] Comment is polite and clear about what should not be done.  👍🏾